platform/alliance-branding-plugin/pom.xml

```diff
@@ -66,8 +62,6 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Export-Package />
```

Comment:
Did you mean to remove this export-packages tag? Without it this bundle will export all its classes